qa/tasks/cephfs/test_admin.py

```diff
@@ -2659,3 +2659,28 @@ def test_with_health_warn_with_2_active_MDSs(self):
                               errmsgs=health_warn)
         self.run_ceph_cmd(f'mds fail {mds1_id} --yes-i-really-mean-it')
         self.run_ceph_cmd(f'mds fail {mds2_id} --yes-i-really-mean-it')
+
+
+class TestFSSet(TestAdminCommands):
+
+    def test_fs_set_max_mds_when_unhealthy_without_confirm(self):
+        '''
+        Test that command "ceph fs set max_mds <num>" without the confirmation
+        flag (--yes-i-really-mean-it) fails when cluster is unhealthy.
```

Comment:
It is `ceph fs set <fs_name> max_mds <num>`

Reply:
True, fixed.